### ISSUE TITLE
Dock Hiding Impossible With Yabai On

### ISSUE THREAD

**Issue by anuragrao04** [koekeishiya/yabai#1951]
I want to completely hide the dock. I came across a workaround to do this on MacOS that goes like this:

```zsh
defaults write com.apple.dock autohide -bool true && killall Dock
defaults write com.apple.dock autohide-delay -float 1000 && killall Dock
defaults write com.apple.dock no-bouncing -bool TRUE && killall Dock
```

It sets the delay for the dock to pop up to an arbitrarily large number. 

Yabai does a very useful thing with the dock where it automatically sets the autohide delay to 0 (unable to verify numerically but it feels like it when using the dock). However, this interferes with my plan to hide the dock completely. Is there an option in Yabai to disable it changing dock settings? I went through the wiki but I could not find anything relevant and any help would be greatly appreciated!

**Comment by koekeishiya** [comment]
Uh, yabai doesn't actually do anything to change the Dock autohide settings, so not sure why this is happening.

**Comment by koekeishiya** [comment]
This is fixed on master in https://github.com/koekeishiya/yabai/commit/1240042a1b5382d7fbfd74f140dce57880c68ed4

**Comment by koekeishiya** [comment]
Fixed in v7.0.0.

**Comment by anuragrao04** [comment]
Thanks! just upgraded and it works flawlessly
